[CAM-12860] docs(additional-batch-permissions): Add specific example (#1130)
camunda/camunda-docs-manual: content/user-guide/process-engine/authorization-service.md

* Create Batch Set Variables
* Create Batch Correlate Messages

> The specific GRANT and REVOKE permissions "Create Batch …" precede over the general "Create" permission.

Reviewer: ```suggestion
GRANT and REVOKE authorizations with "Create Batch …" permissions precede over Create.
```
